packages/layout/src/Carousel.ts

```diff
@@ -57,6 +57,11 @@ export class Carousel extends HTMLWidget {
             ;
     }
 
+    exit(domNode, element) {
+        this.widgets().forEach(w => w.target(null));
```

Comment:
Maybe we should do this at the `Widget.prototype.exit` level for all `widgetArray` properties?

Reply:
Yup - I think I will add a new test to the gallery tests, which ensures every widget can be "relocated" without throwing an error and try there?